scapy/contrib/ospf.py

```diff
@@ -367,6 +373,38 @@ class OSPF_NSSA_External_LSA(OSPF_External_LSA):
     type = 7
 
 
+class OSPF_Link_Scope_Opaque_LSA(OSPF_BaseLSA):
+    name = "OSPF Link Scope External LSA"
+    type = 9
+    fields_desc = [ShortField("age", 1),
+                   OSPFOptionsField(),
+                   ByteField("type", 9),
+                   IPField("id", "192.168.0.0"),
+                   IPField("adrouter", "2.2.2.2"),
```

Comment:
Could you use addresses from https://tools.ietf.org/html/rfc5737 instead?

Reply:
These are not actually IP addresses but I have changed them anyway